x-pack/test_serverless/api_integration/test_suites/common/alerting/summary_actions.ts

```diff
@@ -64,31 +71,44 @@ export default function ({ getService }: FtrProviderContext) {
       'kibana.alert.consecutive_matches',
     ];
 
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
+      internalReqHeader = svlCommonApi.getInternalRequestHeader();
+    });
+
     afterEach(async () => {
       await supertest
```

Comment:
```suggestion
      await supertestWithoutAuth
```

Reply:
Nope, this is setup / teardown.  But thanks for the strong review!!!